plugins/prometheus_plugin/prometheus_plugin.cpp

```diff
@@ -61,37 +61,72 @@ namespace eosio {
          const prometheus::TextSerializer _serializer;
          std::shared_ptr<Registry> _registry;
 
-         std::vector<std::tuple<Family<Gauge>&, Gauge&, runtime_metric&>> _runtime_metrics;
+
+         std::vector<std::tuple<Family<Gauge>&, Gauge&, const runtime_metric&>> _gauges;
+         std::vector<std::tuple<Family<Counter>&, Counter&, const runtime_metric&>> _counters;
 
          // metrics for prometheus_plugin itself
          std::unique_ptr<prometheus_plugin_metrics> _metrics;
 
          // hold onto other plugin metrics
-         std::shared_ptr<net_plugin_metrics> net_plugin_metrics_ptr;
-
          prometheus_plugin_impl() { }
 
-         void add_plugin_metric(runtime_metric& plugin_metric) {
-            auto& gauge_family = BuildGauge()
-                  .Name(plugin_metric.family)
-                  .Help("")
-                  .Register(*_registry);
+         void add_gauge_metric(const runtime_metric& plugin_metric) {
+               auto &gauge_family = BuildGauge()
+                     .Name(plugin_metric.family)
+                     .Help("")
+                     .Register(*_registry);
```

Comment:
This is an odd re-format of the code. The before looks more like what we would want.